test/lit.cfg.py

```diff
@@ -41,7 +41,7 @@
 
 llvm_config.use_default_substitutions()
 
-llvm_config.use_clang()
+llvm_config.use_clang(use_installed=True)
```

Comment:
Thanks for providing the info about corresponding llorg change. Having looked at it, I think that for us  itmight be better to use `CLANG` env variable as you suggested in your first version of this PR, because GitHub Actions machines already has pre-installed clang and we do some not very pleasant manipulations to remove outdated clang from there and replace with a newer one. By having a possibility to specifically select a desired clang version through specifying a path to it we could cleanup our GA config files.

Reply:
Just to clarify, is the suggestion to avoid purging older Clang versions by setting the explicit CLANG path to `/usr/bin/clang-13`? I could definitely try this, although I haven't quite figured out whether this `CLANG` global variable is working without the LIT parameter...

Reply:
In the event it requires the LIT parameter, we could still keep my LIT changes alongside with replacing purging lines with the environment variable export.

Reply:
If I understand correctly, `use_installed` option means to look into `PATH` instead of build directory and above all that is `CLANG` env variable which can override both. So, from CI point of view this is not needed anymore.

However, for local `out-of-tree` environments it makes sense to pass it, just to avoid adding new env requirements for our downstream users

Reply:
That's been my reasoning as well. I'll make sure to add comments so that all of this is reflected in the code.